api/controller/common.proto

```diff
@@ -8,10 +8,12 @@ import "github.com/mwitkow/go-proto-validators/validator.proto";
 
 // RunningOperation respresents a long-running operation.
 message RunningOperation {
-  // Progress from 0.0 to 1.0; can decrease compared to the previous value.
-  float progress = 1;
+  // Finished steps of the operaion.
+  int32 finished_steps = 1;
   // Text describing the current operation progress step.
   string message = 2;
+  // Total steps needed to finish the operation.
```

Comment:
Can it increase compared to the previous value?

Reply:
Yes, it can

Reply:
> Please add answers to code comments

Reply:
Comment updated.